docs/template-windows.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
+  annotations:
+    description: "OCP kubevirt windows VM template"
+    tags: "kubevirt,ocp,template,windows"
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: windows-vm-${NAME}
```

Comment:
Same reasoning about the name as above - should itname be == to NAME variable or with this prefix?

Reply:
open to any suggestion, not changed for the moment

Reply:
In my opinion it should be exactly equal to `NAME`.

Reply:
same comment as above

Reply:
done